login.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="master-container">
+    <div class="login-container">
+      <img class="wolox-logo"
+        src="https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f08/x/ab567d29d6f06a584a4102325415d39f/LogoWolox.png"
```

Comment:
If the image doesn't exist on the project, download it so you can use it locally